guides/source/active_job_basics.md

```diff
@@ -268,13 +268,6 @@ end
 # on your staging environment
 ```
 
-If you want more control on what queue a job will be run you can pass a `:queue`
-option to `set`:
-
-```ruby
-MyJob.set(queue: :another_queue).perform_later(record)
-```
```

Comment:
This felt more approprite to be at the end of this section. So that the ordering is:
1. `queue_as`
2. `queue_as` with block
3. `set` with `queue`